arm-trafficmanager/2015-11-01/trafficmanager.json

```diff
@@ -0,0 +1,740 @@
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "endpointType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of the Traffic Manager endpoint to be created or updated."
+          },
+          {
+            "name": "endpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be created or updated."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Endpoint"
+            },
+            "description": "The Traffic Manager endpoint parameters supplied to the CreateOrUpdate operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "The created or updated Endpoint.", 
+             "schema": { 
+               "$ref": "#/definitions/Endpoint" 
+             } 
+          } 
+        }
+      },
+      "delete": {
+        "tags": [
+          "Endpoints"
+        ],
+        "operationId": "Endpoints_Delete",
+        "description": "Deletes a Traffic Manager endpoint.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "endpointType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "endpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "", 
+             "schema": { 
+               "$ref": "#/definitions/Endpoint" 
```

Comment:
Are you sure about this? Upon successful deletion of the endpoint, the service gives back info about the deleted endpoint?